### DISCUSSION TITLE
【Feature】Support DifyAI Knowledge Base

### DISCUSSION THREAD

**Discussion by kenneth-bro** [coaidev/coai#286] [Discussions | 交流]
[//]: # (方框内删除已有的空格，填 x 号)
+ [ ] 我已确认目前没有类似 issue
+ [x] 我已确认我已升级到最新版本
+ [x] 我已完整浏览项目 README 和项目文档并未找到解决方案
+ [x] 我理解并愿意跟进此 issue，协助测试和提供反馈
+ [x] 我将以礼貌和尊重的态度提问，不得使用不文明用语 (包括在此发布评论的所有人同样适用, 不遵守的人将被 block)
+ [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**功能描述**
这个能支持dify、coze这样的chat-message 或者工作流的API接口收费吗

**相关截图 (如果有)**

![image](https://github.com/user-attachments/assets/7ebab381-d05f-4ea3-93c4-f4d898d48346)

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
来来来，来 #285 ！

[a comment by an automated account is not shown]
